packages/core/src/providers/DAppProvider.tsx

```diff
@@ -34,7 +36,11 @@ function DAppProviderWithConfig({ children }: WithConfigProps) {
         {readOnlyChainId && readOnlyUrls && (
           <ReadOnlyProviderActivator readOnlyChainId={readOnlyChainId} readOnlyUrls={readOnlyUrls} />
         )}
-        <ChainStateProvider multicallAddresses={multicallAddressesMerged}>{children}</ChainStateProvider>
+        <ChainStateProvider multicallAddresses={multicallAddressesMerged}>
+          <NotificationsProvider>
+            <TransactionProvider>{children}</TransactionProvider>
```

Comment:
add newlines
```ts
<TransactionProvider>
  {children}
 </TransactionProvider>
```

Reply:
I wish I could, but prettier prefers it this way.